eta-company-calendar-frontend/src/app/shared/services/auth.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { tap } from 'rxjs/operators';
+import { AuthResponse } from '../models/auth-response.model';
+import { ApiCommunicationService } from './api-communication.service';
+import { ConfigurationService } from './configuration.service';
+
+@Injectable()
+export class AuthService {
+
+  constructor(private readonly app: ApiCommunicationService,
+              private readonly config: ConfigurationService) { }
+
+  public login(email: string, password: string): Observable<AuthResponse> {
+    return this.app.auth()
+    .requestToken(email, password)
+    .pipe(
+      tap((token) => {
+        this.config.setToken(token);
+      })
+    );
```

Comment:
Ez a pipeolás elvégezhető lenne a service-be is

Reply:
Úgy érted hogy a LoginComponent-ben végezzem el?